### PR TITLE
Page layout for application

### PR DESCRIPTION
**Pull request by sharonwarner**
Connects #9 

- Creates React component that create a similar header/footer to the EDD website
- Adds the header/footer to the application page layout

Things left to do:
1) Add hover functionality for links
1) Add grey box in EDD site's footer

Mobile view:
Screen Shot 2020-04-22 at 2 30 42 PM

Monitor view:
Screen Shot 2020-04-22 at 2 31 39 PM

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,6 +35,7 @@
         "helmet": "^3.21.2",
         "i18next-browser-languagedetector": "^4.0.2",
         "i18next-http-backend": "^1.0.3",
+        "icomoon-react": "^2.0.16",
```
Comment (sharonwarner) [inline package.json:38]: I don't love having this dependency. I'm creating a ticket to work on next week to turn these into regular SVGs. https://github.com/cagov/benefits-screener/issues/50

File: src/client/components/Header/index.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+import React from "react";
+import iconSet from "../../../../public/icons.json";
+
+class Header extends React.PureComponent {
+  render() {
+    return (
+      <header className="header border-bottom border-secondary">
+        <Navbar className="justify-content-between" variant="dark" bg="primary">
+          <Navbar.Brand href="https://ca.gov">
+            <img
```
Comment (kalvinwang) [inline src/client/components/Header/index.js:13]: i see the icons here and in the snapshot but don't see them in your PR screenshot.. why's that?

File: src/client/components/Header/index.js [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+            <Nav.Link href="https://edd.ca.gov/claims.htm">
+              <span className="text-dark">Claims</span>
+            </Nav.Link>
+            <Nav.Link href="https://edd.ca.gov/employers.htm">
+              <span className="text-dark">Employers</span>
+            </Nav.Link>
+            <Nav.Link href="https://edd.ca.gov/newsroom.htm">
+              <span className="text-dark">Newsroom</span>
+            </Nav.Link>
+            <Nav.Link href="https://edd.ca.gov/serp.html?q=">
```
Comment (kalvinwang) [inline src/client/components/Header/index.js:66]: I like how this page is .html and everything else is .htm

File: src/client/App.scss [hunk 4]
```diff
@@ -1 +1,16 @@
-@import "bootstrap/scss/bootstrap";
\ No newline at end of file
+// Any bootstrap variables we're overwriting must be defined before importing bootstrap.
```
Comment (kalvinwang) [inline src/client/App.scss:1]: thanks!

File: src/client/components/Header/index.js (hunk 2 above)
Comment (sharonwarner) [reply src/client/components/Header/index.js:13]: We're leaving out the icons in the white portion of the header, at least for now! We're only including the icons in the blue portion of the header next to "Log In" and "Home".

**Pull request merged**